docs/6.integrator/faq.md

```diff
@@ -213,7 +213,7 @@ Users create accounts with human-readable names (eg `alice`) which can contain m
 
 ### Is there a minimum account balance? {#is-there-a-minimum-account-balance}
 
-To limit on-chain "dust", accounts (and contracts) are charged rent for storing data on the chain. This means that if the balance of the account goes below some `threshold * rent_on_block` then account can be removed by anyone. Also any user can remove their own account and transfer left over balance to another (beneficiary) account.
+To limit on-chain "dust", accounts (and contracts) are charged a refundable deposit for storing data on the chain. This means that if the balance of the account does not have enough balance to cover an increased deposit for additional storage of data, storing additional data will fail. Also, any user can remove their own account and transfer left over balance to another (beneficiary) account.
 
 There will be a restoration mechanism for accounts removed (or slept) in this way implemented in the future.
```

Comment:
I'm not sure if we should leave this here also, causes confusion. We should add the relevant details if/when this mechanism is implemented.

Reply:
I'm confused. Errors are thrown in https://github.com/near/nearcore/blob/279044f09a7e6e5e3f26db4898af3655dae6eda6/runtime/runtime/src/lib.rs#L451-L739 and https://github.com/near/nearcore/blob/83b1f80313ec982a6e16a1941d07701e46b7fc35/runtime/runtime/src/verifier.rs#L75-L222 whenever there is not enough balance for storage. What do you mean?

Reply:
I meant this part sorry
``` There will be a restoration mechanism for accounts removed (or slept) in this way implemented in the future. ```

Reply:
Ah great! Will remove then 👍